### ISSUE TITLE
OpenApi: Error while running the server

### ISSUE THREAD

**Issue by kaushikb11** [jina-ai/examples#164]
Syntax Error is raised when you run `openapi_server`. This is due to the template being added by the Jina Dev Bot at the top of the file. As `__future__` imports must be before all code-relevant lines, even those that have no effect. Also, to note the files are auto-generated using OpenAPI Spec.

>     from __future__ import absolute_import
> SyntaxError: from __future__ imports must occur at the beginning of the file

Also, I was wondering if we could add a FastAPI example with multiple endpoints as mentioned in the OpenAPI Spec for Jina Flow. Plus, FastAPI has Swagger UI for interactive exploration, to call and test the API directly from the browser. Would like to take this task up, if it is required.

**Comment by nan-wang** [comment]
@kaushikb11 Thanks for your comments! Would you please elaborate more on the issue and add more code snippets? Plus, we'd love to hear more about your idea to adapt FastAPI-style to jina.

**Comment by kaushikb11** [comment]
@nan-wang Absolutely!

Context: We aim to run the Server generated using [this](https://github.com/jina-ai/examples/blob/master/openapi/jinaai.yaml) Swagger Spec in the `examples/openapi/` directory.
#### Expected UI for interactive exploration, to call and test the API directly from the browser.
![jina_openapi2](https://user-images.githubusercontent.com/45285388/91247174-4c8e4600-e76f-11ea-81d2-2af1ed920f4e.png)

#### Here are the steps to reproduce the mentioned issue:
In the `examples` root directory.
1. `cd openapi/`
2. `pip3 install -r requirements.txt`
3. `python3 -m openapi_server`

#### We face the issue at this point
![jina_openapi](https://user-images.githubusercontent.com/45285388/91246870-975b8e00-e76e-11ea-8a7f-ac6b6992f673.png)

#### What caused the Issue?
TLDR; `__future__` imports need to be at the top.
>Syntax Error is raised when you run openapi_server. This is due to the template being added by the Jina >Dev Bot at the top of the file. As __future__ imports must be before all code-relevant lines, even >those that have no effect. Also, to note the files are auto-generated using OpenAPI Spec.

The temp minor fix I did to resolve to issue by removing/commenting the template code.
```
__copyright__ = "Copyright (c) 2020 Jina AI Limited. All rights reserved."
__license__ = "Apache-2.0"
```
---

### FastAPI
As you could notice there's no use-case example associated with the `openapi` example like `flower-search` or `hello-world` for Users to experiment with Jina. Will explore Jina's Rest API Spec (https://docs.jina.ai/chapters/restapi/index.html) and see if something could be built over it using FastAPI. 
The idea is to have a RestAPI example and have the Abstraction with which Backend(any?) Engineers could get started with Jina. Users could use FastAPI's Swagger UI to interact, call, and test the API directly from the browser with params while a Flow is set on `block()`. Let me know what you think of it and is it something that's required?

**Comment by JoanFM** [comment]
Closing due to inactivity

Jina now leverages FastAPI via daemon, https://docs.jina.ai/master/chapters/remote/run-remote-pod-console.html
